README.md

```diff
@@ -12,21 +12,15 @@ TripleA is an open source gaming community, free to play, 100% open source and v
 - ***Contact-us***: 
   - [Forums: Questions & Help](https://forums.triplea-game.org/category/10/help-questions)
   - [Bug Tracker](https://github.com/triplea-game/triplea/issues/new)
-  - [Gitter](https://gitter.im/triplea-game/social)
 - ***Technical Documentation***: 
-  - [Developer Setup](https://github.com/triplea-game/triplea/tree/master/docs/dev)
-  - [Project Documentation](https://github.com/triplea-game/triplea/tree/master/docs/)
+  - Map Making, Admin, and Developer Docs are on the [wiki](https://github.com/triplea-game/triplea/wiki)
 
 
-
-- ***Dashboards***
-  - [Is the lobby running?](https://stats.uptimerobot.com/14RYqsN5m)
```

Comment:
Curious why you removed this link?

Reply:
Good question; Ultimately I don't think it's that useful/obvious for the end-users and is perhaps more useful for developers. The link is now on the wiki home page.